lib/stencil-init.js

```diff
@@ -123,6 +132,16 @@ class StencilInit {
             });
         }
 
+        if (!cliOptions.packageManager) {
+            prompts.push({
+                type: 'list',
+                name: 'packageManager',
+                message: 'What is your favourite Package Manager?',
```

Comment:
The question itself sounds like a matter of taste. However, if I understood correctly we want to ask `What package manager do you want to use?`.  Sometimes answers on these 2 questions could be different. Just a thought.

Reply:
It is pretty much actually. Even if pnpm is the fastest, it's still a matter of taste. Actually, I took the question from Catalyst, so I think should be fine for now